pg_search/src/postgres/storage/utils.rs

```diff
@@ -41,6 +41,17 @@ impl BM25Page for pg_sys::Page {
         Some(T::from(PgItem(item, (*item_id).lp_len() as pg_sys::Size)))
     }
 
+    unsafe fn read_pg_item(&self, offno: OffsetNumber) -> Option<PgItem> {
+        let item_id = pg_sys::PageGetItemId(*self, offno);
+
+        if (*item_id).lp_flags() != pg_sys::LP_NORMAL {
+            return None;
+        }
+
+        let item = pg_sys::PageGetItem(*self, item_id);
+        Some(PgItem(item, (*item_id).lp_len() as pg_sys::Size))
+    }
+
```

Comment:
This isn't necessary.  There's `Page::read_item<T>()` that already exists and should be used